addons/website_event_sale/controllers/main.py

```diff
@@ -12,7 +12,6 @@ class WebsiteEventSaleController(WebsiteEventController):
 
     @route()
     def event_register(self, event, **post):
-        event = event.with_context(pricelist=request.website.id)
```

Comment:
And I'm ashamed of how much time it took me to find this... :sweat_smile:  Because WHY would I get a pricelist id in the context when I could see there wasn't any pricelist in db? >.<

Reply:
This should be fixed in stable versions.

Reply:
I'll do it in another PR to not delay this PR, thanks =]